llmfoundry/utils/builders.py

```diff
@@ -89,7 +89,10 @@ def build_callback(name: str, kwargs: Dict[str, Any]) -> Callback:
             'log_optimizer_metrics', True),)
     elif name == 'generate_callback':
         prompts = kwargs.pop('prompts')
-        return Generate(prompts=list(prompts), **kwargs)
+        # Generate callback used to be batch_log_interval, so this is for backwards compatibility
+        interval = kwargs.pop(
```

Comment:
won't this be a keyerror?

Reply:
yeah i think you'll want to do
```
kwargs.pop('interval', None) or f"{kwargs.pop('batch_log_interval')}ba"
```
or else the first one will KeyError